### PR TITLE
Hotfixes

### PR DESCRIPTION
**Pull request by Zeeerooo**
Changes:

- Database commit only made after accept all the events (avoiding duplication)
- Solve build_id size error
- Add carriers dynamically, with test

### CODE REVIEW

File: app/api/__init__.py [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-
 from app import app
+from flask_restful import Api
 
 # API stuff
 
 api = Api(app)
 app.config['BUNDLE_ERRORS'] = True
-from . import registration_view
 from . import events_view
```
Comment (Phfollert) [inline app/api/__init__.py]: from <package> import events_view

File: app/api/events_view.py [hunk 2]
```diff
@@ -71,30 +71,39 @@ def read_events_from_file():
...
 
-def read_events(jsonvar, device, sim, app_version_code):
+def read_and_save_events(jsonvar, device, sim, app_version_code):
     total_events = 0
     del jsonvar["device_records"]
     del jsonvar["sim_records"]
 
     for events_name, events in jsonvar.items():
-        total_events += save(events_name, events, device, sim, app_version_code)
-    app.logger.info("Eventos Almacenados: " + str(total_events))
```
Comment (Phfollert) [inline app/api/events_view.py]: change it to english

File: app/models/carrier.py [hunk 3]
```diff
@@ -24,7 +24,22 @@ def __repr__(self):
...
+    @staticmethod
+    def get_carrier_or_add_it(args):
+        """
+        Search a carrier and retrieve it if exist, else create a new one and retrieve it.
+        """
+
+        carrier = Carrier.query.filter(Carrier.mnc == args["mnc"], Carrier.mcc == args["mcc"]).first()
+        if not carrier:
+            carrier = Carrier(mnc=args["mnc"], mcc=args["mcc"], name="Unknown")
+            db.session.add(carrier)
```
Comment (Phfollert) [inline app/models/carrier.py]: do you write db.session.commit() after the add?

Comment (Zeeerooo) [inline app/models/carrier.py]: yepi

**Comment by dmadariaga** [comment]
LGTM

File: app/models/sim.py [hunk 4]
```diff
@@ -4,43 +4,47 @@
 
 
 class Sim(base_model.BaseModel):
-    '''
+    """
     Clase tarjeta sim.
```
Comment (sebablasko) [inline app/models/sim.py]: spanish?

**Comment by Phfollert** [comment]
LGTM

**Pull request merged**